test/e2e-go/cli/goal/expect/goalDryrunRestTest.exp

```diff
@@ -26,13 +26,14 @@ proc TestGoalDryrunExitCode { DRREQ_FILE TEST_PRIMARY_NODE_DIR EXPECTED_STATUS_C
     expect {
         timeout { ::AlgorandGoal::Abort "goal clerk dryrun-remote timeout" }
         $EXPECTED_MESSAGE {puts "message matched"; set MESSAGE_MATCHED 1; close}
-        eof
-    }
-    catch wait result
-    set STATUS_CODE [lindex $result 3]
-    if { $STATUS_CODE != $EXPECTED_STATUS_CODE } {
-        puts "Exit code: $STATUS_CODE, expected: $EXPECTED_STATUS_CODE"
-        ::AlgorandGoal::Abort "Progran exited with incorrect code"
+        eof {
+            catch wait result;
+            set STATUS_CODE [lindex $result 3];
+            if { $STATUS_CODE != $EXPECTED_STATUS_CODE } {
+                puts "Exit code: $STATUS_CODE, expected: $EXPECTED_STATUS_CODE"
+                ::AlgorandGoal::Abort "Progran exited with incorrect code"
+            }
```

Comment:
Should the check for proper STATUS code be placed outside of the EOF condition, in case EOF is not reached.

Reply:
done